### ISSUE TITLE
Prebid Server renderer validation only checks adUnit.renderer

### ISSUE THREAD

**Issue by muuki88** [prebid/Prebid.js#6792]
## Type of issue

Bug

## Description

The validation in prebid server only checks for `adUnit.renderer`.

https://github.com/prebid/Prebid.js/blob/0c8a30820f25d06a400e75fcff1de676dd6c42f1/modules/prebidServerBidAdapter/index.js#L597

However it is possible to define a renderer on the `mediaType` property as described in [the outstream docs](https://docs.prebid.org/dev-docs/show-outstream-video-ads.html#renderers).

## Steps to reproduce

Define an ad slot with outstream context

```javascript
pbjs.addAdUnit({
    code: 'video1',
    mediaTypes: {
        video: {
            context: 'outstream',
            playerSize: [640, 480],
            renderer: {
                url: 'example.com/videoRenderer.js',
                render: function (bid) { renderVideo(...)  }
            }
        }
    }
});
```

Configure prebid server integration. 

### Expected results

If  only `adUnit.mediaTypes.video.renderer` is defined the request should be sent to prebid server

### Actual results

If only  `adUnit.mediaTypes.video.renderer` is defined the request is not sent to prebid server.

## Platform details

- Prebid.js 4.39.0

**Comment by patmmccann** [comment]
@muuki88 it seems like you know exactly what to write in the pr?

**Comment by muuki88** [comment]
Hopefully 😁 Will open one once I'm back from vacation

**Comment by patmmccann** [comment]
Fixed on #6896
